tests/TestCase/View/Helper/ThumbHelperTest.php

```diff
@@ -100,6 +102,52 @@ private function _image($filename = 'test.png'): int
         return (int)$response['data']['id'];
     }
 
+    /**
+     * Create image and media stream for test.
+     * Return id
+     *
+     * @param string $filename the File name.
+     * @return array|null The image Data.
+     */
+    private function _imageData($filename = 'test.png'): ?array
+    {
+        $apiClient = ApiClientProvider::getApiClient();
+
+        $filepath = sprintf('%s/tests/files/%s', getcwd(), $filename);
+        $response = $apiClient->upload($filename, $filepath);
+
+        $streamId = $response['data']['id'];
+        $response = $apiClient->get(sprintf('/streams/%s', $streamId));
+
+        $type = 'images';
+        $title = 'The test image';
+        $attributes = compact('title');
+        $data = compact('type', 'attributes');
+        $body = compact('data');
+        $response = $apiClient->createMediaFromStream($streamId, $type, $body);
+        $response['id'] = (int)$response['data']['id'];
+
+        return $response;
+    }
+
+    /**
+     * Initialize Thumb Helper test
+     *
+     * @return void
+     */
+    public function testInitialize(): void
+    {
+        $this->Thumb = new ThumbHelper(new View());
+        $actual = $this->Thumb->getConfig('cache');
+        $expected = 'default';
+        static::assertEquals($expected, $actual);
+
+        Cache::setConfig('cache', 'default');
```

Comment:
in this second case, test should try another config, not the default one.

Reply:
```
Cache::setConfig('cache', '_cake_core_');
$actual = $this->Thumb->getConfig('cache');
$expected = 'default';
static::assertEquals($expected, $actual);
```

This config can be right?

Reply:
or like this:` Cache::setConfig('cache', 'test');`